08_arrays/main.go

```diff
@@ -43,45 +43,213 @@ func main() {
 	// E.g.: [1, 2, 3] => ["x", "xx", "xxx"]
 	// Print the new array.
 
+	// create an array with 10 integers, fill it
+	// create emtpy array that will be filled with "x"
+	// loop to the value of intArray and add x as many times as the value to lengthIntArray
+	// print result
+	var intArray []int
+	intArray = append(intArray, 1, 2, 3, 4, 5, 6, 7, 8, 9, 9)
+	// debug
+	fmt.Println("Print intArray:", intArray)
+
+	// create new empty array of 10 strings
+	var lengthIntArray []string
+	// counter
+	var arrayAssemblyCounter = 0
+
+	// loop to create output array
+	for {
+		// if counter is > than the last/[10] array element break the loop
+		if arrayAssemblyCounter > intArray[arrayAssemblyCounter]-1 {
+			break
+		}
+		// append "x" to the array, by calling the function with a loop that returns the concat'd string
+		lengthIntArray = append(lengthIntArray, concatVariable(intArray[arrayAssemblyCounter]))
+		// debug
+		fmt.Println("intArray element:", intArray[arrayAssemblyCounter])
+		// increase counter to run with the next array element
+		arrayAssemblyCounter++
+	}
+	// print result array, task req, DONE
+	fmt.Println(lengthIntArray)
+
 	// TODO: Create 2 arrays with 10 integers.
 	// Create 3 more arrays:
 	// - One with the sum of the elements of the 2 arrays.
 	// - One with the difference of the elements of the 2 arrays.
 	// - One with the multiplication of the elements of the 2 arrays.
 	// E.g. [1, 2, 3] and [4, 5, 6] => [5, 7, 9], [-3, -3, -3], [4, 10, 18]
-	// Print the 3 new arrays.
+	//
+	//	// Print the 3 new arrays.
+
+	// create int array with 10 elements
+	var anotherIntArray []int
+	// fill it
+	anotherIntArray = append(anotherIntArray, 1, 4, 5, 4, 6, 8, 5, 13, 5, 12)
+	// create another
+	var yetAnotherIntArray []int
+	// fill it
+	yetAnotherIntArray = append(yetAnotherIntArray, 23, 4, 1, 1, 3, 5, 6, 4, 6, 2)
+
+	// create array for sums of element of the 2 arrays
+	var sumArray []int
+	// create array for difference of elements of the 2 arrays
+	// var diffArray []int
+	// // create array for multiplication of elements of the 2 arrays
+	// var multArray []int
+
+	// create loop for element sum calculation
+	// counter
+	var sumCounter int = 0
+	for {
+		if sumCounter > 9 {
+			break
+		}
+		// append sum of element of arrays
+		sumArray = append(sumArray, anotherIntArray[sumCounter]+yetAnotherIntArray[sumCounter])
+		// increase counter
+		sumCounter++
+	}
+	// print result sumArray
+	fmt.Println(sumArray)
+
+	// close main function, to be able to create function
+}
+
+// math array functions below
+// sum of array elements function
+// counter
+// var mathFunctionsCounter = 0
+// // var sumResult int = 0
+
+// func sumElements(arrayElementOne, arrayElementTwo) {
+// 	if mathFunctionsCounter > arrayElementTwo-1 {
+// 		break
+// 	}
+// 	return arrayElementOne + arrayElementTwo
+// }
+
+// intArray function below
+// create empty variable for "x" string
+var arrayString string = ""
+
+// counter
+var concatCounter = 0
+
```

Comment:
Those variables need to be in the function.

Reply:
It did seem to run like this, but moved them!